_posts/2024-04-27-fcmps.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Full-Chain Membership Proofs Development
+summary: A CCS to develop Full-Chain Membership Proofs has been successfully funded.
+tags: [crypto, community]
+author: Luke "Kayaba" Parker
+---
+
+Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganzations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
```

Comment:
```suggestion
Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.yewtu.be/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganzations](https://www.yewtu.be/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
```

Reply:
NACK, unless this is policy. Several Invidious incidents recently stopped working as Youtube cracks down on adblocking services, so I'd rather point to the canonical source.